schematool/handlers.go

```
@@ -7,69 +7,108 @@ import (
+	return &req, true
+}
 
-	log.Printf("Received /generate-schema-code request in schematool: %+v", req)
-
+func generateSchemaPayload(req SchemaRequest) (map[string]string, error) {
 	goCode, err := GenerateGoSchemaCode(req)
 	if err != nil {
-		log.Printf("Error generating Go schema code: %v", err)
-		http.Error(w, fmt.Sprintf("Error generating schema code: %v", err), http.StatusInternalServerError)
-		return
+		return nil, fmt.Errorf("Error generating schema code: %w", err)
```

golangci-lint (2.11.4)

[error] ST1005: error strings should not be capitalized

(staticcheck)